lib/routes/token.js

```diff
@@ -11,21 +12,107 @@ const config = require('../config');
 const db = require('../db');
 const encrypt = require('../encrypt');
 const logger = require('../logging')('routes.token');
+const P = require('../promise');
+const Scope = require('../scope');
 const validators = require('../validators');
 
+const MAX_TTL = config.get('expiration.accessTokenS');
+const AUTHORIZATION_CODE = 'authorization_code';
+const REFRESH_TOKEN = 'refresh_token';
```

Comment:
nit: suggest naming these `AUTHORIZATION_CODE_GRANT` or similar so it's a bit more obvious the context in which they'll be used.